charts/rancher-cis-benchmark-crd/6.1.0-rc1/templates/clusterscanbenchmark.yaml

```
@@ -0,0 +1,54 @@
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  name: clusterscanbenchmarks.cis.cattle.io
+spec:
+  group: cis.cattle.io
+  names:
+    kind: ClusterScanBenchmark
+    plural: clusterscanbenchmarks
+  scope: Cluster
```

Same for this file which should match with https://github.com/rancher/cis-operator/blob/master/crds/clusterscanbenchmark.yaml.

```suggestion
    plural: clusterscanbenchmarks
    singular: clusterscanbenchmark
  scope: Cluster
```

---

done